controllers/argocd/applicationset.go

```diff
@@ -378,8 +416,17 @@ func (r *ReconcileArgoCD) reconcileApplicationSetRoleBinding(cr *argoprojv1a1.Ar
 	}
 
 	if roleBindingExists {
+		if applicationSetDisabled {
+			if err := r.Client.Delete(context.TODO(), roleBinding); err != nil {
```

Comment:
can we have a `log.Info` here that mentions that the applicationSet roleBinding is being deleted because applicationSet was disabled?